config/peerpods/kata-remote.conf

```diff
@@ -13,7 +13,8 @@
 
 [hypervisor.remote]
 remote_hypervisor_socket = "/run/peerpod/hypervisor.sock"
-remote_hypervisor_timeout = 600
+remote_hypervisor_timeout = 900
+disable_guest_selinux = true
```

Comment:
Are these changes related to enabling annotations processing?

Reply:
The kata-remote.conf that's committed here is not matching with the one deployed. Hence I updated it to the latest. I'll also commit the configuration used to create the machineconfig from the original file to avoid any future inconsistency

Reply:
Fair enough, makes sense.  I wasn't sure if the changes are related since they are parts of the same commit.

Reply:
I'll double check as well just to be sure.

Reply:
Using the following cmd on the original file will get the plaintext configuration toml
```
grep source: mc-40-kata-remote-config.yaml | cut -d "," -f2 | base64 -d | gunzip
```